README.md

```diff
@@ -1 +1,56 @@
-# bookstroe-react-app
\ No newline at end of file
+# BookStore
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> It is a Single Page App (SPA) that allows users to see the books that fetched form API and add books.
+
+
+## Built With
+
+- HTML
+- CSS 
+- Javascript
+- linters
+- webpack
+- React
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repository in your local machine 
+- run `npm install` in the terminal.
+- run `npn run start`.
+
+## Live Demo
+
+[Math Magicians live demo]()
+
+## Author
+
+👤 **Mustafa Fahimy**
+
+- GitHub: [@fahimy143](https://github.com/fahimy143)
+- LinkedIn: [Mustafa Fahimy](https://www.linkedin.com/in/mustafa-fahimy-307566236/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/fahimy143/math-magician/issues).
+
+## Show your support
+
+Give a ⭐️ to if you like this project!
+
+
+## Acknowledgments
+
+- Microverse
+- Coding partner
+- Code reviewer
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The link to your MIT license is broken because the file is missing in your project folder. Kindly add it 👍🏼